pkg/iptables/iptables.go

```diff
@@ -254,7 +255,7 @@ func WithClients(v4, v6 Client) ControllerOption {
 // New generates a new iptables rules controller.
 // If no options are given, IPv4 and IPv6 clients
 // will be instantiated using the regular iptables backend.
-func New(opts ...ControllerOption) (*Controller, error) {
+func New(registerer prometheus.Registerer, opts ...ControllerOption) (*Controller, error) {
```

Comment:
Not sure about that one: We could also pass the registerer as an option. And use the default registry or don't register the counters at all if no registerer is supplied.

Reply:
@clive nice!

Reply:
done 😄